build.gradle

```diff
@@ -15,6 +15,7 @@ plugins {
 	id 'org.asciidoctor.convert' version '1.6.1'
 	id 'org.ajoberstar.grgit' version '3.1.1'
 	id "io.spring.nohttp" version "0.0.3.RELEASE"
+	id "io.spring.dependency-management" version "1.0.7.RELEASE" apply false
```

Comment:
`1.0.8.RELEASE` is there already: https://plugins.gradle.org/plugin/io.spring.dependency-management

Reply:
Gary, how about this one to address as well?

Thanks